README.md

```diff
@@ -23,24 +23,25 @@ Please note: We take Terraform's security and our users' trust very seriously. I
 ## Quick Starts
 
 - [Using the provider](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)
-- [Provider development](docs/contributing)
+- [Provider development](https://hashicorp.github.io/terraform-provider-aws/)
```

Comment:
This will actually take you to the landing page of the contributor guide, the fork contents are hosted here: https://breathingdust.github.io/terraform-provider-aws/